docker-compose.yml

```diff
@@ -110,8 +110,8 @@ services:
       - ldap
     ports:
       - "${SMTPPORT:-25}:25"
-      - "465:465"
-      - "587:587"
+      - "${SMTPSPORT:-465}:465"
+      - "${MSAPORT:-587}:587"
```

Comment:
What does MSA stand for? Mail Submission A...?

Maybe a better name would just be $SUBMISSIONPORT ?

Reply:
It stands for _Mail Submission Agent_ (as per https://stackoverflow.com/questions/15796530/what-is-the-difference-between-ports-465-and-587). In [IANA's service name registry](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=587), this is indeed called _submission_ - I'll adjust that.

Reply:
Ah, never heard that abbreviation. IANA also just refers to it as submission, but since this seems to be an accepted term on stackoverflow its probably good here as well.

Thanks for the clarification, will mere now.